src/components/App.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+      <div className="calculator">
+        <Display />
+        <ButtonPanel />
+      </div>
+    </div>
```

Comment:
- [ ] Please wrap the Display and ButtonPanel components in a single React.Fragment short syntax. You want to avoid adding unnecessary DOM nodes.

Reply:
Thank you @mikenath223